asset/quickstart/batch-get-effective-iam-policies/main.go

```diff
@@ -0,0 +1,55 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START asset_quickstart_batch_get_effective_iam_policies]
+
+// Sample asset-quickstart batch get effective iam policies.
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"log"
+
+	asset "cloud.google.com/go/asset/apiv1"
+	assetpb "google.golang.org/genproto/googleapis/cloud/asset/v1"
+)
+
+func main() {
```

Comment:
Seems odd that this is a quickstart. Since it's demonstrating how to make one call, a snippet-type sample seems more appropriate. Is there a reason the samples are structured this way for asset?

Reply:
I think it's just a legacy reason that someone creates the sample for our 1st API under quickstart and then people just follow that. Do you have a suggestion on how to restructure the code repo for better readability? Thanks!

Reply:
Interesting. It's probably a bigger conversation with your tech writer, but I see that for asset all of the samples are in a separate page that is not an intuitive landing place: https://cloud.google.com/asset-inventory/docs/samples . For most products, the client library samples are more integrated with other docs; e.g. for this sample, I would expect it to live on this page: https://cloud.google.com/asset-inventory/docs/view-effective-iam-policies , rather than separately under the samples section in the docs. Other products have the client libraries samples integrated with other means of doing the same task, e.g. [storage](https://cloud.google.com/storage/docs/downloading-objects).

However this just general feedback, not a blocker on this PR.

Reply:
Thanks for the useful feedback. I created b/244273253 internally to discuss with the tech writer.